src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout alwaysWriteExceptions="false" pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%replace{%ex}{[\r\n]{1,2}}{|}%n"/>
```

Comment:
I appreciate that `alwaysWriteExceptions="false"` looks a bit strange, but the info is [here](https://stackoverflow.com/a/47760847)